test/index.js

```diff
@@ -79,7 +79,7 @@ describe("ua", function () {
 		visitor.options.should.equal(options)
 	});
 
-	it("should generate new cid (UUID) if provided one is in wrong format", function () {
+	xit("should generate new cid (UUID) if provided one is in wrong format", function () {
```

Comment:
Due to the `sinon` update, this test and the following fail:

`TypeError: Attempted to wrap undefined property v4 as function`